frontend/src/main/components/Sections/SectionsInstructorTable.js

```diff
@@ -0,0 +1,70 @@
+import SectionsInstructorTableBase from "main/components/SectionsInstructorTableBase";
```

Comment:
When introducing a new react component, we should also make a storybook entry for that new component.  Please do that here.